backoffice/src/Controllers/LetterController.cs

```diff
@@ -71,6 +73,30 @@ public async Task<IActionResult> Edit(Guid id)
     }
 
 
+    [HttpPost]
+    public async Task<IActionResult> AddLetterToDroit(Guid droitId,LetterType type)
+    {
+        var droit = new Droit();
+        try
+        {
+            droit = await _droitService.GetDroitAsync(droitId);
+        }
+        catch ( DroitNotFoundException e )
+        {
+            HandleError(_logger, "Droit not found.", e);
+            return RedirectToAction("Index", "Droit");
+        }
+        var model = new LetterForm(droitId);

```

Comment:
I wonder if it's better to have this using an empty constructor... 


var model = new LetterForm{
DroitId = droit.Id,
Type = type, 
Recipient = droit?.Salvor?.Email 
}

then:

  model.Subject = await _service.GetTemplateSubjectAsync(type);
        model.Body = await _service.GetTemplateBodyAsync(type);